website/static/css/pages/wiki-page.css

```diff
@@ -11,6 +11,15 @@
     padding: 10px;
 }
 
+.osf-panel {
```

Comment:
Make sense. I searched the code, osf-panel is used in two places. I moved shared css to style.css